tests/acceptance/expected-failures-on-OCIS-storage.md

```diff
@@ -335,14 +335,9 @@ File and sync features in a shared scenario
 -   [apiWebdavEtagPropagation1/moveFileFolder.feature:315](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiWebdavEtagPropagation1/moveFileFolder.feature#L315)
 
 #### [Searching sharee with displayname](https://github.com/owncloud/ocis/issues/547)
--   [apiSharees/sharees.feature:32](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiSharees/sharees.feature#L32)
--   [apiSharees/sharees.feature:33](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiSharees/sharees.feature#L33)
--   [apiSharees/sharees.feature:53](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiSharees/sharees.feature#L53)
--   [apiSharees/sharees.feature:54](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiSharees/sharees.feature#L54)
+These tests fail because they contain some configuration steps for oc10 specific configuration.
```

Comment:
I raised issue https://github.com/owncloud/ocis/issues/2354 to look through the remaining test failing test scenarios - see the issue for details.